src/mapper/src/lib/components/map/main.svelte

```diff
@@ -1,67 +1,72 @@
 <script lang="ts">
-    import '$styles/page.css';
-    import '$styles/button.css';
-    import '@hotosm/ui/dist/hotosm-ui';
-    import {
-        MapLibre,
-        GeoJSON,
-        FillLayer,
-        LineLayer,
-        hoverStateFilter,
-        SymbolLayer,
-        NavigationControl,
-        ScaleControl,
-        Control,
-        ControlGroup,
-        ControlButton,
-    } from 'svelte-maplibre';
-    import { polygon } from '@turf/helpers';
-    import { buffer } from '@turf/buffer';
-    import { bbox } from '@turf/bbox';
-    import type { Position, Polygon, FeatureCollection } from 'geojson';
+	import '$styles/page.css';
+	import '$styles/button.css';
+	import '@hotosm/ui/dist/hotosm-ui';
+	import {
+		MapLibre,
+		GeoJSON,
```

Comment:
Are these whitespace changes from an IDE plugin or something? Perhaps a prettier plugin?

Reply:
We should align on a style config to avoid back and forth 👍

Reply:
Yeah its from prettier

Reply:
Right! Strange it's not picking up the prettierrc config under src/mapper! Can you copy the values in the plugin somehow?